solutions/swb-reference/integration-tests/tests/isolated/costCenter/create.test.ts

```diff
@@ -26,6 +34,41 @@ describe('Cost Center negative tests', () => {
 
   afterAll(async () => {
     await setup.cleanup();
+    await paabHelper.cleanup();
+  });
+
+  describe('authorization test:', () => {
+    test('ITAdmin can create Cost Center', async () => {
+      const response = await itAdminSession.resources.costCenters.create(validCreateRequest);
+
+      expect(response.status).toEqual(201);
+    });
+
+    test('ProjectAdmin cannot create CostCenter', async () => {
+      try {
+        await pa1Session.resources.costCenters.create(validCreateRequest);
+      } catch (e) {
+        checkHttpError(
+          e,
+          new HttpError(403, {
```

Comment:
Let's call this error up top in the describe section once and reference it everywhere:
```const unauthorizedHttpError = new HttpError(403, { error: 'User is not authorized' });```